articles/overview/what-is-quantum-computing.md

```diff
@@ -0,0 +1,54 @@
+---
+title: What is quantum computing?
+description: Learn what quantum computing is, and what a quantum computer can do
+author: natke
+ms.author: nakersha
+ms.date: 10/14/2019
+ms.topic: article
+uid: microsoft.quantum.overview.what
+---
+
+# What is quantum computing?
+
+Quantum computing is the application of quantum physics to the field of computer science.
+
+The theory of quantum physics posits that matter, at a quantum level is simultaneously in multiple states. And those many states interfere with each other like waves in a tide pool.
+
+Quantum computing stores information in quantum states of matter and uses quantum operations to compute on that information, by harnessing and learning to program quantum interference.
+
+Quantum computing might sound daunting, but with the right resources you can start building quantum applications today.
+
+## The qubit
+
+In quantum computing, a quantum bit is a unit of quantum information, like a classical bit. Where classical bits hold a single binary value such as a 0 or 1, a qubit can hold both values at the same time. This is called **superposition**.
+
+The act of **measurement** changes a qubit. It goes from being in the infinite states of superposition to one of the binary states of 0 or 1. Again, measurement of a qubit is unlike a classical bit, which when 0 is measured as 0, and when 1 is measured as 1.
```

Comment:
```suggestion
The act of **measurement** changes a qubit. It goes from being in a superposition of different states to one of the binary states of 0 or 1. Again, measurement of a qubit is unlike a classical bit, which when 0 is measured as 0, and when 1 is measured as 1.
```
The qubit is not in infinite states of superposition but just in one superposition. And if the qubit is good, which means that only has two possible states, then the superposition is only of two states and not infinitely many.